README.md

```diff
@@ -5,6 +5,19 @@ The template used by `npx @react-native-community/cli init` to bootstrap a React
 [![Build Status][build-badge]][build] [![Version][version-badge]][package] [![MIT License][license-badge]][license] [![PRs Welcome][prs-welcome-badge]][prs-welcome]
 
 
+# Create Latest CLI Project
+Using this command create latest react-native project with templete
+```
+npx @react-native-community/cli init TestApp
```

Comment:
```suggestion
npx @react-native-community/cli@latest init TestApp
```

Reply:
What I mean is that people should know how to install the older version of the project.

The latest project is already written on the official documentation of React Native.